MobileOrg.xcodeproj/project.pbxproj

```diff
@@ -124,6 +124,8 @@
 		83B1312C234AD16D0043D955 /* CloudKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 83B1312B234AD16D0043D955 /* CloudKit.framework */; };
 		CE67A65B23A5CDD900B2B0BD /* SwiftyDropbox.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CE67A65923A5CB6500B2B0BD /* SwiftyDropbox.framework */; };
 		CE67A65F23A5CE1800B2B0BD /* Alamofire.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CE67A65E23A5CE1800B2B0BD /* Alamofire.framework */; };
+		F02A539B245F8B1E005A1948 /* linkkey@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = F02A5399245F8B1E005A1948 /* linkkey@2x.png */; };
+		F02A539C245F8B1E005A1948 /* datepick@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = F02A539A245F8B1E005A1948 /* datepick@2x.png */; };
```

Comment:
Do not forget to remove these resources if you agree with my proposal about emojis.

Reply:
I agree.  done.